### PR TITLE
Fill out PVC pages

### PR DESCRIPTION
**Pull request by spadgett**
I noticed these pages didn't have much detail when trying to troubleshoot a problem.

screen shot 2018-06-29 at 10 58 12 am

screen shot 2018-06-29 at 11 16 27 am

/assign @rhamilto

### CODE REVIEW

File: frontend/public/components/persistent-volume-claim.jsx [hunk 1]
```diff
@@ -22,11 +24,31 @@ spec:
...
 const menuActions = [Cog.factory.ModifyLabels, Cog.factory.ModifyAnnotations, Cog.factory.Edit, Cog.factory.Delete];
 
+const PVCStatus = ({pvc}) => {
+  const phase = pvcPhase(pvc);
+  if (!phase) {
+    return '-';
+  }
+
+  switch (phase) {
+    case 'Pending':
```
Comment (jhadvig) [inline frontend/public/components/persistent-volume-claim.jsx]: Could we Enum these phases ?

File: frontend/public/components/factory/list.jsx [hunk 2]
```diff
@@ -120,7 +120,16 @@ const listFilters = {
...
     return types.selected.has(type) || !_.includes(types.all, type);
-  }
+  },
+
+  'pvc-status': (phases, pvc) => {
+    if (!phases || !phases.selected || !phases.selected.size) {
+      return true;
+    }
+
+    const phase = _.get(pvc, 'status.phase');
```
Comment (jhadvig) [inline frontend/public/components/factory/list.jsx]: Isn't the phase always present?

File: frontend/public/components/persistent-volume-claim.jsx (hunk 1 above)
Comment (spadgett) [reply frontend/public/components/persistent-volume-claim.jsx]: Can't add enums in jsx :/ Needs TypeScript

**Comment by spadgett** [comment]
@jhadvig thanks, updated

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/public/components/persistent-volume-claim.jsx (hunk 1 above)
Comment (jhadvig) [reply frontend/public/components/persistent-volume-claim.jsx]: Ohh.... :(

**Comment by spadgett** [comment]
Switched to hourglass icon to be consistent with routes.

**Comment by jhadvig** [comment]
@spadgett thanks !
/lgtm

**Pull request merged**